Package.swift

```diff
@@ -48,8 +48,6 @@ let package = Package(
       ],
       path: "GoogleDataTransport",
       exclude: [
-        "CHANGELOG.md",
-        "README.md",
```

Comment:
I see there is a comment on [line 41](https://github.com/google/GoogleDataTransport/pull/20/files#R41) above this target's declaration to "Restructure directory structure to simplify the excludes here." 

Perhaps adding a folder `Tests/` at root to contain all the test directories. Additionally, maybe `ProtoSupport/` could be moved to root as well. This would eliminate the need for the `excludes` section entirely. Thoughts?

Reply:
Migrating to the structure described at https://github.com/firebase/firebase-ios-sdk/blob/master/AddNewPod.md would be good. Any changes would need to be coordinated with google3 and copybara